src/test/java/io/confluent/connect/hdfs/avro/TopicPartitionWriterTest.java

```diff
@@ -16,22 +16,29 @@
 
 import org.apache.hadoop.fs.FileStatus;
 import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+import org.apache.kafka.common.TopicPartition;
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.Struct;
 import org.apache.kafka.connect.sink.SinkRecord;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
 import org.junit.Test;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.TimeUnit;
 
+import io.confluent.common.utils.MockTime;
+import io.confluent.connect.hdfs.DataWriter;
```

Comment:
Seems like there are a lot of extra imports here that aren't used in the changed lines.

Reply:
Thanks, `src/test` is not covered by checkstyle, so this is still a manual process.